hw4/src/GrowListTest.java

```diff
@@ -0,0 +1,21 @@
+import java.awt.List;
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class GrowListTest {
+    GrowList<String> g = new GrowList<String>();
+
+    // Testing the rep-invariant; correct SDL needs to be in place.
```

Comment:
Should we say here that it is not possible to break the rep invariant and not break the contract?

Reply:
yeah, I think there is a good spot to put that comment. Other than that I think its ready to submit